wireless_modules/boot.py

```diff
@@ -0,0 +1,34 @@
+
+import esp
+import network
+import config
+
+
+def do_connect(essid, password):
+    """
+    Continuously tries to connect to the WIFI
+    :param essid: The WIFI name to connect to, in string format
+    :param password: The WIFI password to connect, in string format
+    :return: Nothing
+    """
+    print("Attempting to connect to WIFI")
+    # Create station interface
+    station_interface = network.WLAN(network.STA_IF)
+    station_interface.active(True)
+
+    # Connect to WiFi
+    station_interface.connect(essid, password)
+
+    # Wait for the module to connect
+    while not station_interface.isconnected(): 
+        pass
+    
+    print("Connected, network config:", station_interface.ifconfig())
+
+
+# Allows the code to be uploaded with tools such as rshell / ampy
+esp.osdebug(None)
+do_connect(config.ESSID, config.PASSWORD)
```

Comment:
It would be really great if we could load all of the files in one go by using a shell script. There would be much nicer ways of doing this but something like this. (From the middle_module folder)
```
ampy -p $PORT -b 115200 put ../config.py
echo "main.py loaded"
ampy -p $PORT -b 115200 put main.py
echo "main.py loaded"
ampy -p $PORT -b 115200 put boot.py
echo "boot.py loaded"


# ... and so on for all the files
```

Reply:
I also think that if `boot.py` is in the global middle_module folder `config.py` should be as well as they can be used for all the modules.

Reply:
As a team, do you think we should generally use `rshell` or `ampy`? I though `rshell` was what we were using.

Reply:
Apparently ampy is much easier to use and most widely used from what I have seen around the web